packages/web-components/src/components/cbp-icon/cbp-icon.tsx

```diff
@@ -95,6 +95,11 @@ export class CbpIcon {
       viewbox: "0 0 512 512",
       path: "M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zm0-384c13.3 0 24 10.7 24 24V264c0 13.3-10.7 24-24 24s-24-10.7-24-24V152c0-13.3 10.7-24 24-24zM224 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z",    
     },
+    "external-link-alt": {
```

Comment:
Please update the icon story (control dropdown) with any icons you've added (in this PR and recently). It needs to reflect this list.